src/sample_diff.rs

```diff
@@ -0,0 +1,224 @@
+pub const DIFF: &[u8; 8715] = b"\
```

Comment:
Shall we use one of the example diffs used in the README, or something else? I was going to say we should choose a shortish one, but I can see the attraction in using a diff long enough to not fit on the screen -- i.e. that it prevents the user from seeing two themes on the screen at the same time.

Reply:
👍 I switched to https://github.com/vuejs/vue/commit/7ec4627902020cccd7b3f4fbc63e1b0d6b9798cd, from the README.